io-classes/io-classes.cabal

```
@@ -6,13 +6,13 @@ license:             Apache-2.0
 license-files:
   LICENSE
   NOTICE
-copyright:           2019 Input Output (Hong Kong) Ltd.
+copyright:           2019-2023 Input Output (Hong Kong) Ltd.
```

Thanks for spotting this, it actually should be:
```suggestion
copyright:           2019-2023 Input Output Global Inc (IOG)
```